core/proto/core.proto

```diff
@@ -22,6 +22,7 @@ message ValidationRecord {
 
         repeated string authorities = 5;
         optional string url = 6;
+        repeated bytes addressesTried = 7; // net.IP.MarshalText()
```

Comment:
Sure - would here be the best place or would it make more sense to add the comment where the structure is defined?